README.md

```diff
@@ -36,6 +40,14 @@ adres:
 http://localhost:3000/
 ```
 
+### GO <3
```

Comment:
I see you want to express your love for GO on the dev branch xd

Reply:
[❤️](https://emojipedia.org/red-heart)[❤️](https://emojipedia.org/red-heart)[❤️](https://emojipedia.org/red-heart)